include/lsst/meas/modelfit/detail/polynomials.h

```diff
@@ -34,8 +34,8 @@ namespace lsst { namespace meas { namespace modelfit { namespace detail {
  * vectors evaluates the polynomial (or computes a derivative).
 */
 template <int N>
-struct Vandermonde {
```

Comment:
This doesn't seem to belong to this commit.

Reply:
Good point. I made it a separate commit with a suitable commit message